docs/architecture/overview.md

```diff
@@ -21,12 +21,3 @@ This proxy is written in Go. Check more details about the implementation in this
 ### Apprepository CRD and Controller
 
 Chart repositories in Kubeapps are managed with a `CustomResourceDefinition` called `apprepositories.kubeapps.com`. Each repository added to Kubeapps is an object of type `AppRepository` and the `apprepository-controller` will watch for changes on those type of objects to update the list of available charts to deploy.
-
-### `chart-repo`
```

Comment:
we probably need to maintain some docs saying that `chart-repo` and `chartsvc` are not here, why and point to the their documentation